ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/StatementExecutor.java

```diff
@@ -286,6 +232,46 @@ private void executeStatement(
     completeStatusFuture(commandId, successStatus);
   }
 
+  private void handleRunScript(Command command) throws Exception {
+    if (command.getStreamsProperties().containsKey(DdlConfig.SCHEMA_FILE_CONTENT_PROPERTY)) {
+      String queries =
+          (String) command.getStreamsProperties().get(DdlConfig.SCHEMA_FILE_CONTENT_PROPERTY);
+      List<QueryMetadata> queryMetadataList = ksqlEngine.buildMultipleQueries(false, queries,
+                                          command.getStreamsProperties());
+      for (QueryMetadata queryMetadata : queryMetadataList) {
+        if (queryMetadata instanceof PersistentQueryMetadata) {
+          PersistentQueryMetadata persistentQueryMetadata = (PersistentQueryMetadata) queryMetadata;
+          persistentQueryMetadata.getKafkaStreams().start();
+        }
+      }
+    } else {
+      throw new KsqlException("No statements received for LOAD FROM FILE.");
+    }
+  }
+
+  private String handleCreateAsSelect(final CreateAsSelect statement,
+                                      final Command command,
+                                      final CommandId commandId,
+                                      final Map<Long, CommandId> terminatedQueries,
+                                      final String statementStr) throws Exception {
+    QuerySpecification querySpecification =
+        (QuerySpecification) statement.getQuery().getQueryBody();
+    Query query = ksqlEngine.addInto(
+        statement.getQuery(),
+        querySpecification,
+        statement.getName().getSuffix(),
+        statement.getProperties(),
+        Optional.empty()
```

Comment:
I think this should be `statement.getPartitionByColumn()`.

Reply:
Yep - thanks!